File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.cs
public class Solution {
   public  int[] ProductExceptSelf(int[] nums)
   {
       int[] res = new int[nums.Length];
       int i = 0;
       int len = nums.Length;
       while (i < len)
       {

           int total = 1; 
           for (int j = 0; j < nums.Length; j++)
           {
               if (i == j)
                   continue;
                else
               {
                  total = total * nums[j];

               }
           }
           res[i] = total;
           ++i;
       }

       return res;
   }
}

File: Data Structures & Algorithms/number-of-sub-arrays-of-size-k-and-average-greater-than-or-equal-to-threshold/submission-1.py
class Solution:
    def numOfSubarrays(self, arr: List[int], k: int, threshold: int) -> int:
        res=0
        l , n = 0 , len(arr)
        curSum=arr[l]
        if k==1 and curSum>=threshold:
            res+=1
        for r in range(1,n):
            
            if r - l + 1 > k:
                curSum-=arr[l]
                l+=1
            curSum+=arr[r]
            if (r - l + 1 == k):
                if int(curSum/k) >= threshold:
                    res= res+1

            
            
        return res
